Tools/autotest/arducopter.py

```diff
@@ -5661,17 +5695,33 @@ def test_alt_estimate_prearm(self):
         self.context_push()
         ex = None
         try:
-            self.set_parameter("EK2_ALT_SOURCE", 2)
+            # disable barometer so there is no altitude source
+            self.set_parameter("SIM_BARO_DISABLE", 1)
+            self.set_parameter("SIM_BARO2_DISABL", 1)
             self.wait_gps_disable(position_vertical=True)
+
+            # turn off arming checks (mandatory arming checks will still be run)
+            self.set_parameter("ARMING_CHECK", 0)
+
+            # delay 12 sec to allow EKF to lose altitude estimate
+            self.delay_sim_time(12)
```

Comment:
We should monitor the EKF flags using `wait_ekf_flags` here.

Reply:
Maybe but the wait_ekf_flags tests all bits are exactly what you expect.. there's only one that we really know about in this test.